.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Library version and API version (please complete the following information):**
+ - Library: [e.g. 10.0.0]
+ - API [e.g. v6]
```

Comment:
should probably ask for versions of dependent libraries since this has come up often. Maybe include the output of `pip list` or `pip freeze`

Reply:
👍🏻